### DISCUSSION TITLE
Incorrect File Upload Path

### DISCUSSION THREAD

**Discussion by ibirman-vbrick** [thunderclient/thunder-client-support#634] [Q&A] [answered]
When I try to add a file using the Files option under Form Fields on a Windows computer, the Path to the file is created as an absolute path from c:\, but the backslashes aren't escaped so it doesn't work.   If I let it create a JavaScript code snippet, it creates the path like this:

formdata.append("", fs.createReadStream("C:\temp\shipit.jpg"));

But it should have double backslashes instead.

Is this a known issue?  If not, should I create an Issue?

**Comment by rangav** [comment, marked as answer]
@ibirman-vbrick please create a new issue, thanks
